doc/api/v8.md

```diff
@@ -271,6 +271,78 @@ buffers and external strings.
 }
 ```
 
+## `v8.getCppHeapStatistics(detailLevel)`
+
+This method returns [CppHeap][] statistics regarding memory consumption and
+utilization. See V8 [`CollectStatistics()`][] API. It returns a
+[`HeapStatistics`][] object. You can learn more about the properties in
```

Comment:
```suggestion
utilization. See V8 [`CollectStatistics()`][] API. It returns an object with a structure similar
to the [`cppgc::HeapStatistics`][] object. You can learn more about the properties in
```

I think this returns `cppgc::HeapStatistics` object instead (which is different from `v8::HeapStatistics`)?